torch/nn/quantized/functional.py

```diff
@@ -427,26 +427,12 @@ def hardtanh(input, min_val=-1., max_val=1., inplace=False):
 
 def hardswish(input, scale, zero_point):
     # type: (Tensor, float, int) -> Tensor
-    r"""Applies the quantized version of the hardswish function, element-wise,
-    as described in the paper:
-
-    `Searching for MobileNetV3`_.
-
-    .. math::
-        \text{Hardswish}(x) = \begin{cases}
-            0 & \text{if~} x \le -3, \\
-            x & \text{if~} x \ge +3, \\
-            x^2/6 & \text{otherwise}
-        \end{cases}
+    r"""This is the quantized version of :meth:`torch.nn.functional.hardswish`.
```

Comment:
probably should be :meth:`~torch.nn.functional.hardswish`

Reply:
it works and is used in pytorch codebase both ways, but sure